R/PlatinumGenomes-QC/rHelpers/setup.R

```diff
@@ -54,7 +89,8 @@ DisplayQueryResults <- function(result, n=6) {
     # Suppress the printing of sample identifiers when the data contains allele
     # values.  This is brittle since the query could use a different name for the columns
     # containing the identifier or allele.
-    if(2 == length(intersect(c("call_call_set_name", "alternate_bases"), colnames(result)))) {
+    if(2 == length(intersect(c("call_call_set_name", "alternate_bases", "alt.alternate_bases"),
+                             colnames(result)))) {
       result <- mutate(result, call_call_set_name = "not displayed")
```

Comment:
Why not just have the following, in order to exclude the column automatically:

```R
result <- subset(result, select=-c(call_call_set_name))
```
It basically performs the following:
```R
> result
  call_call_set_name alternate_bases alt.alternate_bases
1                  1               8                  15
2                  2               9                  16
3                  3              10                  17
4                  4              11                  18
> result <- subset(result, select=-c(call_call_set_name))
> result
  alternate_bases alt.alternate_bases
1               8                  15
2               9                  16
3              10                  17
4              11                  18
> 
```